vscode/snippets/snippets.json

```diff
@@ -0,0 +1,72 @@
+{
+    "lambda": {
+        "prefix": "lambda",
+        "description": "home",
+        "body": "{|${1:arguments}| ${2:body}}"
+    },
+    "var builtin": {
+        "prefix": "var",
+        "description": "\"var\" invocation",
+        "body": "var ${1:name} = ${2:value}"
+    },
+    "if builtin": {
+        "prefix": "if",
+        "description": "\"if\" invocation",
+        "body": [
+            "if (${1:condition}) {",
```

Comment:
The `( )` around the conditions is not part of the `if` command. When you see something like:

```elvish
if (not $foo) {
  ...
}
```

The `( )` is an [output capture](https://elv.sh/ref/language.html#output-capture) expression.